scm-core/src/main/java/sonia/scm/repository/Repository.java

```diff
@@ -352,6 +352,10 @@ public int hashCode() {
 
   @Override
   public String toString() {
+    return namespace + "/" + name + " (" + id + ")";
```

Comment:
`String.format`?

Reply:
Second thought was to cache this, but you know ... cache invalidation...